packages/proxy/lib/http/response-middleware.ts

```diff
@@ -485,11 +485,13 @@ const CopyCookiesFromIncomingRes: ResponseMiddleware = async function () {
     return this.next()
   }
 
-  this.serverBus.once('cross:origin:automation:cookies:received', () => {
+  const origin = cors.getOriginPolicy(this.req.proxiedUrl)
```

Comment:
I think when it comes to `getOriginPolicy` for cookies, we need to factor in the subdomain. I think this method was originally intended for the `document.domain` matching to prevent AUT reloads, but we might need a different method here all together, as well as rename this method. I am going to try some sorts of a rename effort in #23728

Reply:
No longer using the origin here, so this shouldn't be a concern.